packages/mobile-top-nav/src/media-slider.js

```diff
@@ -1,14 +1,73 @@
-import { LitElement, html, css } from 'lit-element';
+import { LitElement, html } from 'lit-element';
+import mediaSliderCss from './css/media-slider';
+import menus from './data/menus';
+import './wayback-search';
+import './more-slider';
 
 class MediaSlider extends LitElement {
+  constructor() {
+    super();
+    const defaultLinks = { iconLinks: [], featuredLinks: [], links: [] };
+
+    this.mediaSliderOpen = false;
+    this.mediaSliderAnimate = false;
+    this.selectedMenuOption = 'texts';
+
+    // Begin properties not monitored by LitElement
+    this.links = menus[this.selectedMenuOption] || defaultLinks;
+    this.templates = {
+      web: () => html`<wayback-search></wayback-search>`,
+      more: () => html`<more-slider></more-slider>`,
+    };
+  }
+
   static get properties() {
     return {
       mediaSliderOpen: { type: Boolean },
       mediaSliderAnimate: { type: Boolean },
-      links: { type: Array },
+      selectedMenuOption: { type: String },
     };
   }
 
+  shouldUpdate() {
+    if (menus[this.selectedMenuOption]) {
+      this.links = menus[this.selectedMenuOption];
+    }
+    return true;
+  }
+
+  renderIconLinks() {
+    return this.links.iconLinks.map((link) => (
+      html`<a href="${link.url}"><img src="${link.icon}" />${link.title}</a>`
+    ));
+  }
+
+  renderLinks(category) {
+    return this.links[category].map((link) => html`<li><a href="${link.url}">${link.title}</a></li>`);
+  }
+
+  renderSubnav() {
+    const template = this.templates[this.selectedMenuOption];
+
+    if (template) {
+      return template();
+    }
+
+    return html`
+      <h3>${this.links.heading}</h3>
+      <div class="icon-links">
+        ${this.renderIconLinks()}
+      </div>
+      <ul>
+        ${this.renderLinks('featuredLinks')}
+      </ul>
+      <h4>Top</h4>
+      <ul>
+        ${this.renderLinks('links')}
+      </ul>
```

Comment:
I would structure your `renderSubnav` a bit differently and may even consider pulling the subnav out to be its own component. It looks like there's a decent amount of logic going on in it.
```
get subnav() {
  const template = this.templates[this.selectedMenuOption];
  return template ? template() : this.defaultSubnav;
}

get defaultSubnav() {
    return html`
       <h3>${this.links.heading}</h3>
       <div class="icon-links">
       ...
       ${this.featuredLinks} <--- so it doesn't get executed immediately, keeps it reactive
       ${this.nonFeaturedLinks}
    `;
}

get featuredLinks() {
  return this.renderLinks('featuredLinks');
}

get nonFeaturedLinks() {
  return this.renderLinks('links');
}

renderLinks(category) {
   return this.links[category].map((link) => html`<li><a href="${link.url}">${link.title}</a></li>`);
}
```